subsys/serialization/transport/rp_ll.c

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
Perhaps @carlescufi has any opinions on this?

Reply:
I get a file from Low Level team from one of their experiments with rpmsg. Based on that I altered it to our needs. Maybe they had it from `rpmsg_nrf53.c` and also altered it. I want to keep it as it is for two reasons:
* Low Level team is planning to create a common abstraction for rpmsg on NRF53, because there will be more users of that e.g. this module and HCI. This file will be removed (maybe also `rpmsg_nrf53.c`) and its functionality will be done by this new module.
* API is different than `rpmsg_nrf53.c`, so I see no point to adjust `rpmsg_nrf53.c` to this API especially as they both probably will be removed/replaced soon.

Reply:
Okay, so this is just a temporary module until we create the proper one?

Reply:
Yes, this module (this file to be exact) is temporary to make serialization up and running without need to wait for Low Level team with their implementation.

Reply:
ok, that's acceptable for the time being then.